execution-engine/engine-test-support/src/internal/utils.rs

```diff
@@ -118,15 +121,19 @@ pub fn create_genesis_config(accounts: Vec<GenesisAccount>) -> GenesisConfig {
         proof_of_stake_installer_bytes,
         standard_payment_installer_bytes,
         accounts,
+        state_infos,
         wasm_costs,
     )
 }
 
-pub fn create_casper_genesis_config(accounts: Vec<GenesisAccount>) -> GenesisConfig {
+pub fn create_casper_genesis_config(
```

Comment:
The purpose of this function is keeping the integration of casperlabs system contracts' test.
So I think it is better not to change this function's interface for future conflicts.

```rust
pub fn create_casper_genesis_config(
    accounts: Vec<GenesisAccount>
) -> GenesisConfig {
```
And we can push the DEFAULT_STATE_INFOS in this function.

```rust
let default = create_genesis_config(accounts, *DEFAULT_STATE_INFOS);
```

Then, we don't have to maintain the tests in engine-tests/src/test/system_contracts

Reply:
Complete to change.
https://github.com/hdac-io/CasperLabs/pull/22/commits/982815a89be3e065345db3f2544e927b57eac272